src/main/java/frc/robot/commands/ScoreCommand.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Robot;
+import frc.robot.RobotContainer;
+import frc.robot.RobotContainer.Teams;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.util.Node;
+
+public class ScoreCommand extends SequentialCommandGroup{
```

Comment:
It's not really needed to do this, I think it's mostly clear how the commands will go